UserService/app/src/main/java/com/bookmyshow/userservice/dao/entity/Permission.java

```diff
@@ -0,0 +1,59 @@
+package com.bookmyshow.userservice.dao.entity;
+
+import javax.persistence.*;
+import java.util.Set;
+
+@Table(name = "PERMISSION")
```

Comment:
Let us follow table name conventions uniformly. We can prefix table name as T_ as used in T_USER